build/wix/main.wxs

```diff
@@ -157,6 +157,18 @@
       <ComponentRef Id="PathUser" />
       <ComponentRef Id="PathSystem" />
     </Feature>
+    <UI>
+       <UIRef Id="WixUI_RD" />
```

Comment:
We already have a `<UIRef>` on line 25; that seems to be a duplicate?
https://github.com/rancher-sandbox/rancher-desktop/blob/0a26306847d0afedb2f0c98cff02c21149159892/build/wix/main.wxs#L25

Reply:
Yes, testing shows it isn't needed.